pyvista/plotting/colors.py

```diff
@@ -974,11 +977,15 @@ def get_cmap_safe(cmap):
             return cmap
         # Else use Matplotlib
         cmap = get_cmap(cmap)
+
     elif isinstance(cmap, list):
         for item in cmap:
             if not isinstance(item, str):
                 raise TypeError('When inputting a list as a cmap, each item should be a string.')
-        from matplotlib.colors import ListedColormap
+        try:
+            from matplotlib.colors import ListedColormap
+        except ImportError:  # pragma: no cover
+            raise ImportError('Listed colormaps require the installation of matplotlib') from None
```

Comment:
You're right. Reverting...

```suggestion
        from matplotlib.colors import ListedColormap
```